mailbox/elasticsearch/pom.xml

```diff
@@ -268,8 +268,8 @@
                     <artifactId>slf4j-api</artifactId>
                 </dependency>
                 <dependency>
-                    <groupId>org.slf4j</groupId>
-                    <artifactId>slf4j-simple</artifactId>
+                    <groupId>ch.qos.logback</groupId>
```

Comment:
a library (or component) should never enforce actual log library.
If you need it for testing purpose, please define a test scope